teuthology/beanstalk.py

```diff
@@ -6,8 +6,9 @@
 import sys
 from collections import OrderedDict
 
+import teuthology.report as report
```

Comment:
Sorry but what's the purpose of doing this? why can't we just ``from teuthology import report``

Reply:
You were right about this; my changes were more heavy-handed than they needed to be! The follow-up PR reverts the unnecessary bits.